src/plugins/coc.js

```diff
@@ -1,28 +1,62 @@
 const { createVerbose } = require('../index')
 
 module.exports = {
+  'cc<=': {
+    priority: -1,
+    fn(a) {
+      const { rand, verbose } = this;
+      const res = rand.next().value % 100 + 1
+      let texts = []
+      let result = ( res <= a )
+      if (res == 1) {
+        result = true
+      }
+      if (res == 100) {
+        result = false
```

Comment:
:eyes: need it.
reason:
* Non-investigator's skill can 100%+
* Hit rate can 100%+